src/main/java/org/swisspush/redisques/handler/RedisquesHttpRequestHandler.java

```diff
@@ -75,15 +71,19 @@ public class RedisquesHttpRequestHandler implements Handler<HttpServerRequest> {
     private final boolean enableQueueNameDecoding;
     private final int queueSpeedIntervalSec;
     private final QueueStatisticsCollector queueStatisticsCollector;
+    private final RedisQuesExceptionFactory exceptionFactory;
     private final QueueStatsService queueStatsService;
     private final GetQueueStatsMentor<RoutingContext> queueStatsMentor = new MyQueueStatsMentor();
 
-    public static void init(Vertx vertx, RedisquesConfiguration modConfig, QueueStatisticsCollector queueStatisticsCollector,
-                            DequeueStatisticCollector dequeueStatisticCollector) {
+    public static void init(
+        Vertx vertx, RedisquesConfiguration modConfig, QueueStatisticsCollector queueStatisticsCollector,
+        DequeueStatisticCollector dequeueStatisticCollector, RedisQuesExceptionFactory exceptionFactory
+    ) {
         log.info("Enabling http request handler: {}", modConfig.getHttpRequestHandlerEnabled());
         if (modConfig.getHttpRequestHandlerEnabled()) {
             if (modConfig.getHttpRequestHandlerPort() != null && modConfig.getHttpRequestHandlerUserHeader() != null) {
-                RedisquesHttpRequestHandler handler = new RedisquesHttpRequestHandler(vertx, modConfig, queueStatisticsCollector, dequeueStatisticCollector);
+                var handler = new RedisquesHttpRequestHandler(
```

Comment:
Congratulations, I think you are the first one in the gateelen&friends universe using _var_ instead of the type :-)

Reply:
Is my assumption correct that this is a request to replace them by explicit type names? :)

Reply:
No it's OK for me :-)